src/ontology/diffs/uberon-diff.md

```diff
@@ -1 +1,4030 @@
+
+#### Added
+- [coronary sinus](http://purl.obolibrary.org/obo/UBERON_0005438) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://cdn.humanatlas.io/digital-objects/ref-organ/blood-vasculature-male/v1.3/assets/3d-vh-m-blood-vasculature.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [coronary sinus](http://purl.obolibrary.org/obo/UBERON_0005438) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://cdn.humanatlas.io/digital-objects/ref-organ/blood-vasculature-female/v1.3/assets/3d-vh-f-blood-vasculature.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+
+### corpus callosum `http://purl.obolibrary.org/obo/UBERON_0002336`
+#### Removed
+- [corpus callosum](http://purl.obolibrary.org/obo/UBERON_0002336) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/Allen_M_Brain.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [corpus callosum](http://purl.obolibrary.org/obo/UBERON_0002336) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/Allen_F_Brain.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+#### Added
+- [corpus callosum](http://purl.obolibrary.org/obo/UBERON_0002336) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://cdn.humanatlas.io/digital-objects/ref-organ/brain-male/v1.4/assets/3d-allen-m-brain.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [corpus callosum](http://purl.obolibrary.org/obo/UBERON_0002336) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://cdn.humanatlas.io/digital-objects/ref-organ/brain-female/v1.4/assets/3d-allen-f-brain.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+
+### cortex of kidney `http://purl.obolibrary.org/obo/UBERON_0001225`
+#### Removed
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_F_Kidney_R.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_M_Kidney_L.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_F_Kidney_L.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+- [cortex of kidney](http://purl.obolibrary.org/obo/UBERON_0001225) [depiction](http://xmlns.com/foaf/0.1/depiction) "https://ccf-ontology.hubmapconsortium.org/objects/v1.2/VH_M_Kidney_R.glb"^^[anyURI](http://www.w3.org/2001/XMLSchema#anyURI) 
+
+#### Added
```

Comment:
Does it mean that will display 4 objects? I think they don't want all of them (we should ask if there's a preffered gender and side?)

Reply:
I understood that they wanted only the latest version but keep the gender and side.